modules/tinymce/src/themes/silver/main/ts/ui/window/SilverInlineDialog.ts

```diff
@@ -58,9 +73,33 @@ const renderInlineDialog = <T>(dialogInit: DialogManager.DialogInit<T>, extra: S
   const dialogEvents = SilverDialogEvents.initDialog(
     () => instanceApi,
     {
-      // TODO: Implement block and unblock for inline dialogs
-      onBlock: () => { },
-      onUnblock: () => { },
+      onBlock: (event) => {
```

Comment:
Given how long this function is, it might be clearer to pull it out as a sibling to the `dialogEvents` const, so that the `onBlock` definition given in `initDialog` is just one or two lines.

Reply:
Addressing a different comment shortened the onBlock implementation quite a bit, which should fix this